acl_setup.php

```diff
@@ -174,6 +181,25 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('Add/Update groups (view ,write)')
+
+ $gacl->add_object('groups', 'Create/Update groups appointment in calender','gcalendar', 10, 0, 'ACO');
+// xl('Create/Update groups appointment in calender (view ,write)')
+
+ $gacl->add_object('groups', 'Group log','glog', 10, 0, 'ACO');
+// xl('Group log (view ,write)')
+
+ $gacl->add_object('groups', 'Group detailed log of appointment in patient record','gdlog', 10, 0, 'ACO');
+// xl('Group detailed log of appointment in patient record (view ,write)')
+
+ $gacl->add_object('groups', 'Send message from the permanent group therapist to the personal therapist','gm', 10, 0, 'ACO');
+ // xl('Send message from the permanent group therapist to the personal therapist (view ,write)')
```

Comment:
Note that the point of the commented out xl is to capture the title for the translation engine, so it needs to be the exact same as the above title.